.github/workflows/deploy-docs.yml

```
@@ -50,5 +50,5 @@ jobs:
         git config --local user.email $GIT_COMMITTER_EMAIL
         git config --local user.name $GIT_COMMITTER_NAME
         git add .
-        git commit --amend --no-edit
+        git commit -m "Updating supplemental resources for ${MIKE_VERSION} documentation deployment"
         git push --force
```

```suggestion
        git push
```

Hopefully no `--force` necessary